XCTWebDriverAgentLib/Categories/XCUIElement+WebDriverAttributes.m

```diff
@@ -108,4 +108,13 @@ - (NSDictionary *)wdSize
       };
 }
 
+- (NSDictionary *)wdLocation
+{
+    return
```

Comment:
For `wdSize ` and `wdLocation` I think it would be better to reuse `wdRect` by calling `return self.wdRect[@"size"]` and `return self.wdRect[@"origin"]`